rise/static/main.js

```diff
@@ -33,14 +33,25 @@ define([
       theme: 'simple',
       transition: 'linear',
       slideNumber: true,
-      start_slideshow_at: 'beginning',
-      /* can be either
-       * "none" - no autoselect,
-       * "first" - select first cell
-       * "code" - select first code cell
+      /* describe where to start slideshow
+       * can be either:
+       * 'beginning' : start on first slide 
+       * 'selected'  : start on current slide
+       * xxx: could be useful to add more policies
```

Comment:
sounds like a good idea for the future

Reply:
In fact, for that case, I mean a hard-wired slide number, maybe `reveal.js` itself offers that possibility, and it would be probably very easy to adapt/implement it in RISE